huxley/www/js/lib/ServerAPI.js

```diff
@@ -34,6 +34,20 @@ var ServerAPI = {
   },
 
   /**
+   * Get a list of all assignments for the given school ID.
+   */
+  getCommitteeAssignments(committeeID) {
+    return _get(`/api/assignments/committee/${committeeID}`);
```

Comment:
IMO we shouldn't make another API endpoint for this. It's really just a filter on the assignments list API. We should then customize the `get` behavior of the AssignmentListView to modify the queryset if any filters are present.

The URI would then become something like `/api/assignments?committee_id=12`.